src/esssans/isis/components.py

```diff
@@ -0,0 +1,44 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2024 Scipp contributors (https://github.com/scipp)
+from typing import NewType
+
+import sciline
+import scipp as sc
+
+from ..types import RawData, RunType
+
+PixelMask = NewType('PixelMask', sc.Variable)
+
+
+class ConfiguredRawData(sciline.Scope[RunType, sc.DataArray], sc.DataArray):
```

Comment:
I'm not sure about the name. Immediately it wasn't very clear what it was meant to represent.
What kind of other configurations are we expecting apart from custom offsets? Should we have a name that is specific to applying offsets?

Reply:
Do you have a suggestion?

Reply:
`RawDataWithComponentUserOffsets`?

Reply:
Should we also rename the provider? I guess `configure_raw_data` made more sense when the output was called `ConfiguredRawData`?